COMMUNITY.md

```diff
@@ -300,4 +300,11 @@ at scale and in ways that can be unexpected for the original implementor.
 As such debugs MUST be guarded in such a way that they can be turned off.
 This PROJECT has the ability to turn on/off debugs from the CLI and it is
 expected that the developer will use this convention to allow control
-of their debugs.
\ No newline at end of file
+of their debugs.
+
+### CLI-Changes
+
+CLI's are a complicated ugly beast.  Additions or changes to the CLI
+should use a DEFUN to encapsulate one setting as much as is possible.
```

Comment:
Just to make it super explicit, perhaps:

"Unrelated CLI definitions should not be bundled together."